core/node/rpc/forwarder.go

```diff
@@ -45,6 +62,10 @@ func allowNoQuorum[T any](req *connect.Request[T]) bool {
 	return req.Header().Get(RiverAllowNoQuorumHeader) == RiverHeaderTrueValue
 }
 
+func shouldFallbackToRemotes(err error) bool {
+	return slices.Contains(shouldFallbackToRemotesErrCodes, AsRiverError(err).Code)
+}
```

Comment:
Error codes are used together with `RiverErrorImpl`. A better place is put this functionality is in the base package next to `RiverErrorImpl`. This is more localised and other parts of the code base can use it when they use `RiverErrorImpl`. Now it's private to the rpc package.

Suggestion: delete `shouldFallbackToRemotesErrCodes` and `shouldFallbackToRemotes`. Introduce `base#IsOperationRetriableOnRemotes(err) bool` that follows the same pattern as some of the existing error checks.